src/main/java/com/base2services/jenkins/SqsProfile.java

```diff
@@ -55,7 +55,12 @@ public String getAWSSecretKey() {
 
     public AmazonSQS getSQSClient() {
         if(client == null) {
-            client =  new AmazonSQSClient(this);
+            if (!"".equals(awsAccessKeyId.trim()) && 
```

Comment:
Hi, thanks for the PR. May I suggest to replace the `!"".equals` with `StringUtils.isNotBlank(awsAccessKeyId)`?